ansible_collections/arista/avd/docs/getting-started/avd-aap.md

```diff
@@ -0,0 +1,326 @@
+
+    - In the new pane, click on `Add`.
+    - A new pane is shown but similar to before, fill in an appropriate `Name` and select `Source from a Project` under `source`.
+    - Under `Source details` click on the search icon under `Project`
+      - Select you newly created project
+    - Under `Source details`, if the `Inventory file` drop down does not show your inventory, feel free to enter it manually.
+    - Under `Update options` make sure `Update on launch` is checked.
+    - Scroll down and click `Save`.
+
+    ![Create source](../_media/getting-started/aap-avd/create-source.png)
+
+=== "Sources - Sync"
+
+    In the new pane, click on `Sync` to update your inventory.
+
+    ![Inventory sync](../_media/getting-started/aap-avd/inv-sync.png)
+
+### View the Inventory
+
+There are a few locations to view the inventory, but for simplicity, we can view it by clicking on `Hosts` on the left pane.
+
+=== "Click on Sync Job"
+
+    - On the left pane, click on `Jobs`.
+      - You should see an inventory sync running or complete.
+    - Click on the inventory sync job.
+
+    ![Inventory job sync](../_media/getting-started/aap-avd/inv-job-sync.png)
+
+=== "Sync job output"
+
+    Towards the bottom of the output, we see that AAP successfully parsed our inventory. In this case, we have loaded nine groups and seven hosts.
+
+    ![Sync job output](../_media/getting-started/aap-avd/inv-job-output.png)
+
+=== "Hosts"
+
+    There are a few locations to view the inventory, but for simplicity, we can take a look by clicking on `Hosts` on the left pane.
+
+    ![View inventory](../_media/getting-started/aap-avd/all-hosts.png)
+
+## Job templates and workflow templates
+
+One thing that may need some clarification is the naming of "job templates." These map to playbooks within our project. There is also an option to build workflow templates: a series of job templates with some control logic built-in. For this example, we will use a job template to build and deploy our node configurations with CVP.
+
+=== "Click on Templates"
+
+    On the left pane, click on `Templates`.
+
+    ![Click on Templates](../_media/getting-started/aap-avd/select-templates.png)
+
+=== "Templates - Add"
+
+    - On the center pane, select `Add`.
+    - In the drop down select `Add job template`.
+
+    ![Add template](../_media/getting-started/aap-avd/add-template.png)
+
+=== "Templates - Job"
+
+    The job template is where we leverage the custom execution environment. Since our setup requires specific Ansible collections and Python packages installed, we would like to use a pre-packaged environment with that software. We can modify a decent number of settings, and they may look familiar from previous history with Ansible configurations. If you need a refresher on these options, please see the [official documentation](https://docs.ansible.com/automation-controller/latest/html/userguide/job_templates.html). Once you are happy with the settings, click `Save`.
```

Comment:
I do agree, but please note the docs are supposed to be consumed via mkdocs, so use the link in the last test from readthedocs (https://ansible-avd--3910.org.readthedocs.build/en/3910/) to preview the generated docs instead of using github.

Reply:
@danieltudares did you try viewing with the PR site build?

Reply:
Yeah the review with readthedocs is better. It's being a while since my last doc review :) all good here, sorry for the noise